src/mkdocs_include_markdown_plugin/event.py

```diff
@@ -634,6 +635,10 @@ def on_page_markdown(
         http_cache: Cache | None = None,
 ) -> str:
     """Process markdown content of a page."""
+    if page.file.src_uri.endswith(".ipynb"):
+        # Jupyter Notebooks may have long outputs to process
+        # (e.g. Plotly figures) that slows down regex string substitutions
+        return markdown
```

Comment:
No, use the global [`exclude` setting](https://github.com/mondeja/mkdocs-include-markdown-plugin?tab=readme-ov-file#exclude) instead. Please, remove this.

Reply:
in order to use that, i had to modify the code further because it can't compute the global settings when the pattern matching is already being computed.

With the latest commit it's now possible to exclude a file that is included in the nav-bar too

Reply:
That's much better than the current implementation.

Reply:
well current implementation actually allows you to exclude some files from the processing. Currently the plugin start working on some files just to cancel everything midway. Why is that better?

I am sorry, but i can't understand what's bad about the current implementation